.github/workflows/main.yml

```diff
@@ -30,3 +30,38 @@ jobs:
         run: python -m pip install -U tox
       - name: Run lint
         run: tox -elint
+  benchmark:
+    runs-on: ubuntu-latest  # windows-latest | macos-latest
+    name: Test changed benchmarks
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2  # OR "2" -> To retrieve the preceding commit.
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v24
+        with:
+          files: |
+            red_queen/games/applications/*.py
+            red_queen/games/mapping/*.py
+          files_ignore: |
+            red_queen/games/applications/__init__.py
+            red_queen/games/mapping/__init__.py
+      - name: Install Deps
+        run: |
+          python -m pip install --upgrade pip
+          pip install .
+      - name: Run every new benchmark
+        run: pytest ${{steps.changed-files.outputs.all_changed_and_modified_files}} -n 2 --verbose --store
+      - name: Print results to text-file
+        run: python -m report.console_tables --storage results/0001_bench.json > results/0001_table.txt
```

Comment:
Do you want to dump this to a file or output it in the logs. I feel like having the output in the CI results will be more useful than downloading a text file of the results table. If you download the results json from the artifact generating the output is trivial but being able to quickly scroll to the benchmarks you care about inspecting is more valuable

Reply:
It was originally dumped in the console logs. I changed it so that it can be downloaded. I will roll back to console logging though, as it is easier for contributors to view.